Sources/StreamChatUI/CommonViews/InputTextView/InputTextView.swift

```diff
@@ -187,4 +192,10 @@ open class InputTextView: UITextView, AppearanceProvider {
             setTextViewHeight()
         }
     }
+
+    private func scrollToCaretPosition(animated: Bool) {
```

Comment:
We are always passing animated: false here, right? Maybe it makes sense to not have this parameter?